woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -125,13 +126,23 @@ def test_categorical_inference(categories):
     if _is_spark_series(categories[0]):
         dtypes = get_spark_dtypes(dtypes)
     for ind, series in enumerate(categories):
-        if ind == len(categories) - 1:
+        if ind >= len(categories) - 2:
```

Comment:
why do we have this change?

Reply:
I was curious but a little unsure, should we add a postal series to this fixture? 

https://github.com/alteryx/woodwork/blob/807ea4fb6fa4ec01db415d483ae0f51ca3e8f23b/woodwork/tests/type_system/conftest.py#L185

Reply:
I don't think we need to add postal to the fixture. You added the test that would test that scenario

Reply:
sounds good!